vault/changelog.release.2021-12-28.md

```diff
@@ -0,0 +1,92 @@
+---
+id: N1Mn5p6O2tVVkdDM0Ys1w
+title: '0.75'
+desc: ''
+updated: 1640719141612
+created: 1640715852170
+---
+
+Dendron 0.75 has sprouted  🌱
+
+This is the last release of the year!
+
+When deleting notes, did you just break some links elsewhere? Now Dendron makes sure to give you a heads up, about any links that will break, with the latest update to the `Dendron: Delete Node` command.
+
+Speaking of broken links, Dendron has an entirely new command: **Convert Link**! Have links you need to update, or links you'd like to otherwise transform into other formats (like usertags or hashtags)? Take a look at [[Convert Link|dendron://dendron.dendron-site/dendron.topic.refactoring#convert-link]] and give it a shot.
```

Comment:
The use cases are not great because a lot of our users don't know about usertags or hashtags. Change to ->

Speaking of broken links, Dendron has an entirely new command: **Convert Link**!  Convert links let you convert broken links to plaintext, replacing the link with the link alias, note name, or text of your choosing. You can also use it to convert existing wiki links into other [[types|dendron://dendron.dendron-site/dendron.topic.refactoring#valid-links]] of wiki links